contracts/governance/GovernanceSettings.sol

```diff
@@ -3,51 +3,99 @@ pragma solidity 0.8.27;
-        return _taskHandlingReward;
+    /// @notice setProposalBurntFee sets the burnt part of fee for a proposal
+    /// @param _proposalBurntFee new proposal burn fee in native tokens
+    function setProposalBurntFee(uint256 _proposalBurntFee) public onlyOwner {
+        _proposalBurntFee = _proposalBurntFee * 1e18;
+        uint256 newProposalFee = _proposalBurntFee + taskHandlingReward + taskErasingReward;
+        // Proposal fee must always be greater than or equal to sum of burntFee and rewards
+        if (newProposalFee > proposalFee) {
+            revert ProposalFeeTooLow(newProposalFee);
+        }
+        proposalBurntFee = _proposalBurntFee;
     }
 
-    /// @notice taskErasingReward is a reward for erasing each task
-    /// @return task erasing reward
-    function taskErasingReward() public pure returns (uint256) {
-        return _taskErasingReward;
+    /// @notice setTaskHandlingReward sets a reward for handling each task
+    /// @param _taskHandlingReward new task handling reward in native tokens
+    function setTaskHandlingReward(uint256 _taskHandlingReward) public onlyOwner {
+        _taskHandlingReward = _taskHandlingReward * 1e18;
+        uint256 newProposalFee = proposalBurntFee + _taskHandlingReward + taskErasingReward;
+        // Proposal fee must always be greater than or equal to sum of burntFee and rewards
+        if (newProposalFee > proposalFee) {
+            revert ProposalFeeTooLow(newProposalFee);
+        }
+        taskHandlingReward = _taskHandlingReward;
+    }
+
+    /// @notice setTaskErasingReward sets a reward for erasing each task
+    /// @param _taskErasingReward new task erasing reward in native tokens
+    function setTaskErasingReward(uint256 _taskErasingReward) public onlyOwner {
+        _taskErasingReward = _taskErasingReward * 1e18;
+        uint256 newProposalFee = proposalBurntFee + taskHandlingReward + _taskErasingReward;
+        // Proposal fee must always be greater than or equal to sum of burntFee and rewards
+        if (newProposalFee > proposalFee) {
+            revert ProposalFeeTooLow(newProposalFee);
+        }
+        taskErasingReward = _taskErasingReward;
+    }
+
+    /// @notice setMaxOptions sets maximum number of options to choose
+    /// @param _maxOptions new maximum number of options
+    function setMaxOptions(uint256 _maxOptions) public onlyOwner {
+        maxOptions = _maxOptions;
     }
 
-    /// @notice maxOptions maximum number of options to choose
-    /// @return maximum number of options
-    function maxOptions() public pure returns (uint256) {
-        return _maxOptions;
+    /// @notice setMaxExecutionPeriod sets the period after the end of voting during which the proposal can be executed
+    /// @param _maxExecutionPeriod new maximum execution period in hours
+    function setMaxExecutionPeriod(uint256 _maxExecutionPeriod) public onlyOwner {
+        maxExecutionPeriod = _maxExecutionPeriod * 1 hours;
```

Comment:
DONT MUTLIPLY IN SETTERS - let the user to enter the native value, in seconds.
This makes the api unnecessary non-standard and unnecessary prevents the admin to enter a value not-rounded to hours.